### PR TITLE
feat: new repositories install hooks

### PR DESCRIPTION
**Pull request by svatwork**
As described in https://typicode.github.io/husky/#/?id=install, this script will make sure that developers will also receive the necessary hooks for having features like commit linting.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
     "husky:pre-commit": "lint-staged && npm run spellcheck",
     "lint": "bin/otomi validate-templates",
     "lint:all": "run-p spellcheck lint",
+    "postinstall": "husky install",
```
Comment (Morriz) [inline package.json:62]: I don't think this has any effect after having done "npm install", as that registers the hooks already.

**Review by Morriz — COMMENTED** [review]
I don't think this has any effect

**Comment by Morriz** [comment]
@svatwork can you keep working on this? I see your suggestions are based on the new v5, where they don't use auto-install We are using v4 configuration (and install I presume). I wouldn't mind if this PR is used to upgrade to v5 and add the exclusion logic to NOT install the hooks in CI.

**Comment by svatwork** [comment]
Yes, sure.

**Comment by Morriz** [comment]
I just tried a fresh npm install husky and see it installs @v4.3.0. So lets forget it for now.

**Pull request closed without merging**